docs/troubleshooting.md

```diff
@@ -0,0 +1,9 @@
+## Troubleshooting 
```

Comment:
Appreciate your patience with the nitpick, and I should have mentioned it earlier.